modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CustomerPortalSite.path

```diff
@@ -310,6 +310,11 @@
 	<td>//*[contains(@class, 'alert-notifications')]//*[text()='${key_alertText}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ANY</td>
+	<td>//p[contains(text(), 'Support Seat')]//..//*[*[name()='svg'][contains(@class,'lexicon-icon-info-circle')]]</td>
+	<td></td>
+</tr>
```

Comment:
@luanakarla why are we calling "ANY" for this element, since it is related to Support Seat? Do you want to get an icon?

Reply:
Makes sense, I'll switch now.